xrpl_binary_codec/Cargo.toml

```diff
@@ -12,7 +12,13 @@ edition = "2021"
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
 hex = "0.4"
-bs58 = { version = "0.5", features = ["check"] }
-sha2 = "0.10"
-libsecp256k1 = "0.7"
+bs58 = {version = "0.5.0", features = ["check"]}
```

Comment:
Is there a reason you made the version more specific? I intentionally used the more 'flexible' 0.5 here. Is there a problem with that?

Reply:
The versions `"0.5"` and `"0.5.0"` both mean `^0.5.0` (https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio). The recommendation is to always use full digits: https://doc.rust-lang.org/cargo/reference/resolver.html#recommendations in order to specify the correct range of versions. Here it may not be important, but if we depended on features in "0.5.1" it would.